packages/core/src/components/axes/ruler.ts

```diff
@@ -24,10 +24,17 @@ export class Ruler extends Component {
 		domainValue: number;
 		originalData: any;
 	}[];
+	isRulerEnabled =  Tools.getProperty(
```

Comment:
this should go inside `render`, that way it's value will update with future renders